### PR TITLE
DP-8854 Footer logo

### PR DESCRIPTION
**Pull request by ygannett**
Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)

## Description

The footer portion which is set up as `03-organisms/by-template/footer.twig` has been hardcoded in the layout template, `page.html.twig`, in Drupal and not using Mayflower where it could be used. This has been causing some inefficiency for maintenance.

In this PR, the following are done:

- Update the content in `<footer>` to match Drupal:
     - Add an aria role to `<footer>` tag
     - Add the privacy policy link
     - Change the hard-coded copyright year to the twig markup to print the current year
- Updates the Google translate JS in the template to match one in the Drupal
- Add a twig block for the navigation to set up the both implementations for Mayflower and Drupal, so the navigation is rendered properly in both environments.

Associated Drupal PR: [DP-8854 Footer #2290](https://github.com/massgov/mass/pull/2290)
If this PR is released before Drupal one, it doesn't affect Drupal side since it has its own hard-coded markup in its themeing.  However, releasing Drupal change without the Mayflower change would cause Releasing this Drupal change without the Mayflower change would cause the footer navigation to be missing.

## Related Issue / Ticket

- [DP-8854 Footer logo](https://jira.state.ma.us/browse/DP-8854)

## Steps to Test

1. Go to any pages with the footer.
2. Check for any difference for the footer display against the prod.
3. Check the copyright year is current.
4. Check the navigation links work as they are supposed to.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

### CODE REVIEW

**Comment by rbayliss** [comment]
Hey @ygannett - Actually, can we move the Google Translate markup out of this template and into `styleguide/source/_meta/_01-foot.twig`?  It seems really weird to me that we're adding javascript along with a template.

**Comment by ygannett** [comment]
@rbayliss The Google translate JS has been moved to`styleguide/source/_meta/_01-foot.twig`.  Can you take a look?

File: styleguide/source/_patterns/03-organisms/by-template/footer.twig [hunk 1]
```diff
@@ -1,48 +1,41 @@
...
+        </a>
+      </div>
+      <div class="ma__footer__social">
+        {% set socialLinks = footerSocialLinks %}
+        {% include "@molecules/social-links.twig" %}
+      </div>
+      <div class="ma__footer__copyright">
+        <p><b>&copy; {{ "now"|date("Y") }} Commonwealth of Massachusetts.</b></p>
+        <p>Mass.Gov&#x00AE; is a registered service mark of the Commonwealth of Massachusetts.</p>
+        <a href="/privacypolicy">Mass.gov Privacy Policy</a>
```
Comment (rbayliss) [inline styleguide/source/_patterns/03-organisms/by-template/footer.twig]: Can we use a block for the privacy policy URL too? Something like `<a href="{{ privacyPolicy | default('/privacypolicy') }}">Mass.gov Privacy Policy</a>`

File: styleguide/source/_patterns/03-organisms/by-template/footer.twig [hunk 2]
```diff
@@ -1,48 +1,41 @@
...
-    })();
-  </script>
-{% endif %}
+<footer class="ma__footer js-footer" id="footer" role="contentinfo">
+  <div class="ma__footer__container">
+    {{ page.footer }}
+  {% block footer_navigation %}
+    {% if page.footer_navigation %}
+      {{ page.footer_navigation }}
+    {% else %}
```
Comment (rbayliss) [inline styleguide/source/_patterns/03-organisms/by-template/footer.twig]: This isn't quite what I had in mind.  I don't think we should be bringing Drupal variables back to Mayflower.  See my comments on https://github.com/massgov/mass/pull/2290#pullrequestreview-121577430 for what I had in mind.

**Review by rbayliss — CHANGES_REQUESTED** [review]
Hey Yukiko, sorry I didn't catch this earlier this week.  It's been a crazy one.

**Comment by ygannett** [comment]
@rbayliss Some changes are made reflecting your comments. Please take a look again.

**Comment by rbayliss** [comment]
Awesome.  This looks great. The Drupal implementation looks 100% identical, except that the "G" in Mass.gov is capitalized in the new version.  I think that's intended though, since it's always looked that way in Mayflower.  I'll merge as soon as this goes green.

**Review by rbayliss — APPROVED** [review]

**Review by mrossi113 — APPROVED** [review]
👍 LGTM!

**Pull request merged**